### PR TITLE
Post ToB

### PR DESCRIPTION
**Pull request by godzillaba**

### CODE REVIEW

File: .github/workflows/build-test.yml [hunk 1]
```diff
@@ -28,9 +28,6 @@ jobs:
       - name: Install packages
         run: yarn
 
-      - name: Check format
-        run: forge fmt --check
-
```
Comment (gzeoneth) [inline .github/workflows/build-test.yml:33]: why?

File: README.md [hunk 2]
```diff
@@ -1,49 +1,72 @@
...
-3. Token bridge retryable(s) redeemed: tokens land in the `L2Forwarder` address (which may or may not have code yet)
-4. Factory retryable redeemed: `L2ForwarderFactory.callForwarder`
-    1. Create and initialize the user's `L2Forwarder` via `Clone` if it does not already exist.
-    2. Call `L2Forwarder.bridgeToL3(...)`
-5. `L2Forwarder.bridgeToL3`
-    1. Send the forwarder's entire token balance through the bridge to L3. The contract's entire ETH (or fee token) balance minus execution fee is sent as submission fee in order to forward all the extra ETH (or fee token) to L3.
+## Deployment Procedure
+Deploy `L1Teleporter`  on any L1, passing in predicted `L2Forwarder`  and `L2ForwarderFactory`  arguments. Deploy `L2ForwarderContractsDeployer` using a generic CREATE2 factory to the same address on any Arbitrum L2’s.
 
-### Retryable Failures and Race Conditions
```
Comment (gzeoneth) [inline README.md:30]: why is this section removed?

File: .github/workflows/build-test.yml (hunk 1 above)
Comment (godzillaba) [reply .github/workflows/build-test.yml:33]: we already have the separate format check

File: README.md (hunk 2 above)
Comment (godzillaba) [reply README.md:30]: i just thought it was too much :shrug: 

added it back in

Comment (gzeoneth) [reply README.md:30]: kinda agree, how about add to a separate markdown file?

Comment (godzillaba) [reply README.md:30]: sounds good to me

File: docs/info.md
Comment (gzeoneth) [inline docs/info.md:1]: also include a link from README to this would be good

**Pull request merged**